### DISCUSSION TITLE
which dongles support install codes ?

### DISCUSSION THREAD

**Discussion by monkeydriven** [Koenkk/zigbee2mqtt#15415] [Devices]
Hi,

to me it is unclear which dongles support "install codes" in zigbee2mqtt. 
I did search but couldn´t find a list or something. 
All I can say is that the conbee II does not support install codes...

Any specific recommendation ?

**Comment by Koenkk** [comment]
Adapters running the Texas Instruments CC2652/CC1352 chips support install codes.

**Reply by kirovilya** [reply]
and ezsp adapter support :)

**Reply by Koenkk** [reply]
I forgot, yes of course!
